Added information about precompresssed files (#6707)
umbraco/UmbracoDocs: 15/umbraco-cms/fundamentals/setup/upgrading/version-specific/README.md

If you have custom C# code that references the models this will also not build. You can either comment out your custom code temporarily until the models have been rebuilt or fix the models manually. To fix the models manually you need to find and replace `IPublishedSnapshotAccessor` with `IPublishedContentTypeCache`.

**Handling Precompressed Files**

> When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco but is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.

Author: ```suggestion
When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco. It is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```

> To disable this behavior, you can set `<CompressionEnabled>false</CompressionEnabled>` in your project file. If you are using Umbraco's templates - `dotnet new umbraco`, this setting is already applied.

Author: ```suggestion
To disable this feature, you can set `<CompressionEnabled>false</CompressionEnabled>` in your project file. If you are using Umbraco's templates - `dotnet new umbraco`, this setting is already included.
```

If you have custom C# code that references the models this will also not build. You can either comment out your custom code temporarily until the models have been rebuilt or fix the models manually. To fix the models manually you need to find and replace `IPublishedSnapshotAccessor` with `IPublishedContentTypeCache`.

**Handling Precompressed Files**

> When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, resulting in additional `.br` and `.gz` files. This behavior is not specific to Umbraco. It is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.

Reviewer: ```suggestion
When upgrading from Umbraco 14 to 15, you might notice that `JavaScript` and `CSS` files are automatically precompressed, adding additional `.br` and `.gz` files. This behavior is introduced in ASP.NET Core version 9, where static files are fingerprinted and precompressed by default at build and publish time.
```
